controller.go

```diff
@@ -162,26 +170,26 @@ func loadDurationFromEnv(varName string, dest *time.Duration) error {
 }
 
 func usage() {
-	fmt.Fprintf(os.Stderr, "usage: %s [options]\n", os.Args[0])
-	fmt.Fprintln(os.Stderr, "where options can be:")
+	log.Errorf("usage: %s [options]\n", os.Args[0])
+	log.Error("where options can be:")
```

Comment:
I would not use the logger for this as this is more of a CLI help message. Would keep it as is.